app/views/warehouse_reports/cas/non_hmis_clients/_client.haml

```diff
@@ -0,0 +1,23 @@
+- candidate = client.last
+%tr
+  %td= candidate[:first_name]
+  %td= candidate[:last_name]
+  - ssn = candidate[:ssn]
+  %td= ssn(ssn)
+  %td= candidate[:date_of_birth]
+  %td
+    - exact_match_id = Cas::NonHmisClient.exact_match(candidate, @report)
+    - if exact_match_id.present?
+      = link_to 'Exact Match', client_path(exact_match_id)
+      = " (#{exact_match_id})"
+      %td
+    - else
+      - partial_match_ids = Cas::NonHmisClient.partial_matches(candidate, @report)
+      - if partial_match_ids.present?
+        %ul.list-unstyled.mb-0
+          - partial_match_ids.each do |partial_match_id|
+            %li
+              = link_to @report.db_names[partial_match_id], client_path(partial_match_id)
+              = " (#{partial_match_id})"
+        %td
+          = f.input :client_id, label: false, required: false, input_html: { name: "clients[#{client.first}]", style: 'width: 8em;', placeholder: 'Client ID' }
```

Comment:
This input should show up even if we have no matches to allow them to manually search and add.